File: NevernoteAPI/Models/Note.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace NevernoteAPI.Models
{
    public class Note
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string LastUpdated { get; set; }
        public string Tags { get; set; }
        public bool? IsFavorite { get; set; }
    }
}

File: NevernoteAPI/ClientApp/src/app/modules/workspace/Models/Note.ts
export class Note {
  Id: number;
  title: string;
  description: string;
  lastUpdated: string;
  isFavorite: boolean;
  tags: string[];

  constructor(
    Id: number,
    title: string,
    description: string,
    lastUpdated: string
  ) {
    this.Id = Id;
    this.title = title;
    this.description = description;
    this.lastUpdated = lastUpdated;
    this.isFavorite = false;
    this.tags = null;
  }
}
